main.go

```diff
@@ -28,6 +29,13 @@ import (
 	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
 )
 
+var _ store.Storer = DatsetAPIStore{}
```

Comment:
I am guessing this is just to check that the store implements the storer interface? If so, a small optimisation to not allocate any memory while still doing the check: `var _ store.Storer = (*DatsetAPIStore)(nil)`